packages/good-design/src/hooks/useModalState.ts

```diff
@@ -0,0 +1,13 @@
+import { useCallback, useState } from "react";
```

Comment:
see modal concept draft here: https://github.com/GoodDollar/GoodWeb3-Mono/pull/27#pullrequestreview-1182907541

important: should be discussed with @sirpy

Reply:
@roma-hladilka 
I saw you merged @sirpy 's modal implementation
but you still use your own
please fix this and we will bet able to merge

Reply:
done